tutor/src/store/offering.ts

```diff
@@ -1,10 +1,17 @@
 import { createSlice, createEntityAdapter } from '@reduxjs/toolkit'
 import { useSelector } from 'react-redux'
+import { union } from 'lodash'
 import { bootstrap } from './bootstrap'
 import { Offering, Course, ID } from './types'
+import UiSettings from 'shared/model/ui-settings'
 
 const offeringAdapter = createEntityAdapter<Offering>({
-    sortComparer: (a, b) => a.default_course_name.localeCompare(b.default_course_name),
+    sortComparer: (a, b) => {
+        //send an offering that is not available to the bottom of the list
+        if(!a.is_available && !a.is_preview_available) return 1
+        //if the number is higher, it is low priority
+        return b.number.localeCompare(a.number)
```

Comment:
Ok, this needs to be extracted to an external function and a unit spec written for it so we can fully test it.
 
I think we'd decided that only non available should appear at the bottom, and preview available should be sorted normally.    But if both offerings are not available we need to compare them to each other, and no need to use localeCompare since we're sorting by a number.
 
It'll need to be something like this, but I'd expect this to have some bug, hence the need for spec.
```
if ( a.is_available == !b.is_available) return Number(a.number)-Number(b.number);
if (a.is_available && !b.is_available) return -1;
return 1
```